internal/wasm/premissioncache.go

```diff
@@ -71,6 +71,9 @@ type permissionCache struct {
 
 	cache  map[string]*permissionCacheEntry
 	logger logging.Logger
+
+	// For tests
+	triggerChan chan struct{}
```

Comment:
IMO, this's overkill for test only or it's a wrong design.

Reply:
Yeah, we can make the tests better following @mathetake 's https://github.com/envoyproxy/gateway/issues/5735#issuecomment-2831679819 later in another PR. This is a quick/dirt fix to unblock the github flows for v1.4.